src/websocket_service/mod.rs

```diff
@@ -72,6 +72,10 @@ impl WebsocketService {
                 }
                 wsclient::RelayClientEvent::Error(e) => {
                     warn!("Received error from relay: {}", e);
+                    while let Err(e) = self.connect().await {
+                        warn!("Error reconnecting to relay: {}", e);
```

Comment:
```suggestion
                        error!("Error reconnecting to relay: {}", e);
```

Reply:
The regular reconnection was already handled, this is only about sudden ones